planning/mission_planner/include/mission_planner/mission_planner_interface.hpp

```diff
@@ -0,0 +1,40 @@
+// Copyright 2023 TIER IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef MISSION_PLANNER__MISSION_PLANNER_INTERFACE_HPP_
+#define MISSION_PLANNER__MISSION_PLANNER_INTERFACE_HPP_
+
+#include <rclcpp/qos.hpp>
+
+#include <autoware_planning_msgs/srv/set_pose_with_uuid_stamped.hpp>
+#include <std_srvs/srv/trigger.hpp>
+
+namespace mission_planner
+{
+
+struct SetMrmGoal
+{
+  using Service = autoware_planning_msgs::srv::SetPoseWithUuidStamped;
+  static constexpr char name[] = "~/srv/set_mrm_goal";
```

Comment:
@isamu-takagi  
It is ok for me, but I saw the comment from @mitsudome-r in https://github.com/autowarefoundation/autoware.universe/pull/3184#discussion_r1159484479

> MRM is abbreviation of "minimum risk maneuver" so I prefer the name to be ~~"/srv/set_mrm_goal"~~ or "/srv/set_emergency_goal"

set_mrm_goal is also ok?

Reply:
set_mrm_goal is okay with me.
I actually wrote the following and didn't notice that two '~' was converted to strikethrough
> "\~/srv/set_mrm_goal" or "\~/srv/set_emergency_goal"

Reply:
I just edited the original comment as well.